### ISSUE TITLE
Pa11y with axe automated testing

### ISSUE THREAD

**Issue by dmundra** [bbc/accessibility-news-and-you#57]
Hello,

I have a proof of concept setup where I have pa11y with axe testing your Jekyll site for accessibility issues. You can see the results here https://github.com/dmundra/accessibility-news-and-you/pull/1, let me know if you want me to share the changes back to this repo as a pull request. Looks like the majority of issues are related to contrast.

Another example of the automation is on the CivicActions accessibility site that you can see in action here: https://github.com/CivicActions/accessibility/pull/128

Let me know if you have any questions about the setup.

Daniel

**Comment by greenc05** [comment]
Yeah thanks @dmundra I'm afraid the Jekyll default styling theme/template used fails on colour contrast. There is a new look and feel on the most recent pages of the site such as https://bbc.github.io/accessibility-news-and-you/guides/accessibility-swarms.html which is fully colour complaint, this new look and feel will be used on all pages when they are updated.
